assets/src/components/form/select/model-selects/status.js

```diff
@@ -0,0 +1,60 @@
+/**
+ * Internal imports
+ */
+import ModelSelect from '../model-select';
+import * as statusModel from '../../../../data/model/status';
+
+/**
+ * External imports
+ */
+import { __ } from '@eventespresso/i18n';
+import { Component } from '@wordpress/element';
+import { PropTypes } from 'prop-types';
+
+/**
+ * Select Component for the Status Model.
+ */
+export default class StatusSelect extends Component {
+	state = {
+		modelName: 'status',
+	};
+
+	static defaultProps = {
+		selectConfiguration: {
+			loadingMessage: () => __( 'Retrieving Statuses.', 'event_espresso' ),
+			noOptionsMessage: () => __( 'No Statuses.', 'event_espresso' ),
+			placeholder: __( 'Select Status...', 'event_espresso' ),
+		},
+		...statusModel.defaultQueryData,
+		getQueryString: statusModel.getQueryString,
+		selectedType: 'registration',
```

Comment:
we can leave this until we see a pattern emerge if you want, but I wonder if `data/model/status/index.js` should include a set of constants for the different status "types" and then you just use the appropriate constant in here (i.e. `STATUS_TYPE_REGISTRATION`)?

Reply:
I like that idea. 
The only downside is that any new statuses added to the db need to be replicated in code, but that's kinda the case anyways, which was never my intent with statuses, although that's how they panned out regardless.